### PR TITLE
Adding week 7 blog post

### PR DESCRIPTION
**Pull request by xtanion**
blog is visible in `blogs.html`

### CODE REVIEW

File: docs/source/posts/2022/2022-08-01-week-7-shivam.rst [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+
+What did you do this week?
+--------------------------
+
+- This week I started with implementing scaling to the animation example.
+
+- I had a meeting with Mohamed in which we discussed the rotation issue, and we found out that glTF uses ``Slerp`` interpolator instead of the ``LinearInterpolator``. Using ``Slerp`` interpolator for rotation fixed the rotation issue.
+
+- Another issue we faced was with the multi-actor system some actors weren't rotating or translating as intended. This was caused because the transformations were applied to the ``polydata`` before creating an actor; Mohamed suggested applying transformation after the actor is created from polydata (keeping the center to origin).
```
Comment (m-agour) [inline docs/source/posts/2022/2022-08-01-week-7-shivam.rst:17]: Maybe separate sentences as `...system. Some...`

File: docs/source/posts/2022/2022-08-01-week-7-shivam.rst [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+
+  .. raw:: html
+
+        <iframe id="player" type="text/html"   width="640" height="360" src="https://user-images.githubusercontent.com/74976752/182166133-585d06ef-55ff-48db-8ce7-98b377ebf8ec.mp4" frameborder="0"></iframe>
+
+
+- Created functions to return a list of animation timelines and apply them to the main timeline for keyframe animations.
+
+- ``CubicSpline`` has not been implemented to glTF animation yet since it works differently than other Interpolators (takes tangent input to smoothen the curve) but It'll be done before our next meeting.
```
Comment (m-agour) [inline docs/source/posts/2022/2022-08-01-week-7-shivam.rst:27]: `it'll` or `it will`

**Review by m-agour — COMMENTED** [review]
Hello @xtanion ,
Please see 2 comments below, thanks.

**Review by ganimtron-10 — COMMENTED** [review]
Hello @xtanion ,
Nice Blog! LGTM.

Also, I had a suggestion, the demo video doesn't look that good, to me it looks like objects are randomly moving. I can't see any interpolation or specific transformation. If you could change or modify the demo video to something else which would clearly show the animation support it would be more attractive.
What's your take on this @m-agour ??

Thanks!

File: docs/source/posts/2022/2022-08-01-week-7-shivam.rst [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+
+- I had a meeting with Mohamed in which we discussed the rotation issue, and we found out that glTF uses ``Slerp`` interpolator instead of the ``LinearInterpolator``. Using ``Slerp`` interpolator for rotation fixed the rotation issue.
+
+- Another issue we faced was with the multi-actor system. Some actors weren't rotating or translating as intended. This was caused because the transformations were applied to the ``polydata`` before creating an actor; Mohamed suggested applying transformation after the actor is created from polydata (keeping the center to origin).
+
+
+  .. raw:: html
+
+        <iframe id="player" type="text/html"   width="640" height="360" src="https://user-images.githubusercontent.com/74976752/182166133-585d06ef-55ff-48db-8ce7-98b377ebf8ec.mp4" frameborder="0"></iframe>
+
```
Comment (skoudoro) [inline docs/source/posts/2022/2022-08-01-week-7-shivam.rst:23]: here, add a link to the Kronos gif to show the comparison.

**Review by skoudoro — COMMENTED** [review]
Hi @ganimtron-10,

Your comment makes sense even if I understand the video. Can you add a link to the original animation from the Kronos GLTF folder @xtanion  on your blogpost? You could also add the gif on the side to show what we were expecting. Thanks!

File: docs/source/posts/2022/2022-08-01-week-7-shivam.rst (hunk 3 above)
Comment (xtanion) [reply docs/source/posts/2022/2022-08-01-week-7-shivam.rst:23]: Sure, I'll add this

**Comment by skoudoro** [comment]
Thank you for the update, merging !

**Pull request merged**